python/ray/serve/_private/deployment_graph_build.py

```diff
@@ -225,13 +226,20 @@ def replace_with_handle(node):
         # TODO: (jiaodong) Need to capture DAGNodes in the parent node
         parent_deployment_node = other_args_to_resolve[PARENT_CLASS_NODE_KEY]
 
+        parent_class = parent_deployment_node._deployment._func_or_class
+        method = getattr(parent_class, dag_node._method_name)
+        if "return" in method.__annotations__:
```

Comment:
I'm not too familiar with `method.__annotations__`. When is this conditional `True` (and `False`)?

Reply:
What are we doing in this conditional's body?

Reply:
For instance, if we define a function `f` with return type `int`:
```
>>> def f() -> int:
            return 5
>>> f.__annotations__
{'return': <class 'int'>}
```
If there is no return type specified, `f.__annotations__` will not contain `return`.

In this conditional, we are including the return type (as a string) in `other_args_to_resolve`, which contains info that will be preserved as the DAG is transformed into a serve executor DAG.

Reply:
I see, that makes sense.